README.md

```diff
@@ -4,3 +4,44 @@ Operator that sets OCP cluster-wide storage defaults.
 Ensures a default storage class exists for OCP clusters, like the [addon-manager](https://github.com/kubernetes/kubernetes/tree/release-1.13/cluster/addons/storage-class) does for kubernetes clusters. Supports AWS and OpenStack. No configuration is required. The created storage class can be made non-default by editing its annotation but cannot be deleted so long as the operator runs.
 
 Will also ensure default CSI volume plugins are installed in a future release when CSI plugins replace in-tree ones (see [csi-operator](https://github.com/openshift/csi-operator)).
+
+# Quick start
+
+Before running the operator manually, you must remove the operator installed by CVO.
+
+```shell
+# Scale down CVO and CSO
+oc scale --replicas=0 deploy/cluster-version-operator -n openshift-cluster-version  
+oc scale --replicas=0 deploy/cluster-storage-operator -n openshift-cluster-storage-operator
+
+# OPTIONAL - delete existing CSO lock
+oc -n openshift-cluster-storage-operator delete lease/cluster-storage-operator-lock
+```
+
+To build and run the operator locally:
+
+```shell
+# Build the operator
+make
+
+# Set kubeconfig and obtain desired version
+export KUBECONFIG=<path-to-kubeconfig>
+export OPERATOR_IMAGE_VERSION=$(oc get clusterversion/version -o json | jq '.status.desired.version' | tr -d "\"")
+
+# Set the environment variables
+export AZURE_DISK_DRIVER_OPERATOR_IMAGE=quay.io/openshift/origin-azure-disk-csi-driver-operator:latest
+export AZURE_FILE_DRIVER_OPERATOR_IMAGE=quay.io/openshift/origin-azure-file-csi-driver-operator:latest
+export AZURE_DISK_DRIVER_IMAGE=quay.io/openshift/origin-azure-disk-csi-driver:latest
+export AZURE_FILE_DRIVER_IMAGE=quay.io/openshift/origin-azure-file-csi-driver:latest
```

Comment:
This wouldn't be enough on non-Azure installations. E.g., in case of AWS, corresponding variables are `AWS_EBS_DRIVER_IMAGE` and `AWS_EBS_DRIVER_OPERATOR_IMAGE`.

Reply:
That is correct, each provider has different variables. The idea was to have at least something as an example, but this can be written better. Listing all possible variables is probably not useful so I changed the doc to describe where those variables can be found with some examples.

Reply:
Fully agreed. Now the doc looks fine. Thank you!